eiffel-syntax-and-usage/the-links-object.md

```diff
@@ -28,18 +28,18 @@ __Description:__ Identifies the flow context of the event: which is the continuo
 
 ### links.activityExecution
 __Type:__ String  
-__Required in:__ [EiffelActivityDequeuedEvent](../eiffel-vocabulary/EiffelActivityDequeuedEvent), 
+__Required in:__ [EiffelActivityCanceledEvent](../eiffel-vocabulary/EiffelActivityCanceledEvent), 
```

Comment:
I guess we should add ".md" to the link even though it did not exist in the old link. Also missing in the following 2 line.